### ISSUE TITLE
Filter results by advisory type

### ISSUE THREAD

**Issue by another-rex** [google/osv-scanner#623]
Add the ability to filter the vulnerability results by the advisory type. 

For example I might be only interested in malicious packages.

**Comment by andrewpollock** [comment]
Is there a canonical definition for "advisory type"?

I wonder if this should be by prefix, in the absence of one?

**Comment by another-rex** [comment]
I don't think there is one, but prefix might be misleading since most prefixes is describing the source rather than the actual type of advisory it is.

**Comment by andrewpollock** [comment]
> prefix might be misleading

I agree, the malicious packages source is the first source that I'm aware of that introduces this notion of there being a type other than a vulnerability advisory...

**Comment by another-rex** [comment]
Closing in favour of a filter by Malicious packages.
